WSSiOS/WSSiOS/Source/Data/Entity/ProfileEntity.swift

```diff
@@ -8,32 +8,36 @@
 import Foundation
 
 struct MyProfileEntity {
-    let nickname, intro, avatarImage: String
+    let nickname, intro: String
```

Comment:
```suggestion
    let nickname, introduction: String
```
네이밍컨벤션 상 줄임말보단 풀워드가 좋을 것 같아서요! 제안해봅니다